universe/Pathfinder.cpp

```diff
@@ -1094,7 +1094,14 @@ std::pair<std::vector<int>, int> Pathfinder::LeastJumpsPath(
 std::pair<std::vector<int>, int> Pathfinder::PathfinderImpl::LeastJumpsPath(
     int system1_id, int system2_id, int empire_id, int max_jumps) const
 {
-    if (empire_id == ALL_EMPIRES) {
+    bool use_system_graph = false;
```

Comment:
This bool seems unnecessary, but I suppose it's fine...

Reply:
using named temporary variables instead of multiple querying is probably a preference coming from my java background (doesnt hurt with booleans in c++)